package.json

```diff
@@ -61,7 +61,7 @@
     "lodash": "^4.17.21"
   },
   "peerDependencies": {
-    "@nestjs/common": "^10.0.0",
+    "@nestjs/common": "^10.3.9",
```

Comment:
@ppetzold it still seems to be updating peer dependencies. Would it also be possible to adjust the repo settings so that I can approve these renovate PRs by myself, to keep up to date and declutter the PR tab